src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java

```diff
@@ -107,13 +110,50 @@ protected void writeColumnSpec(StringBuilder builder, SinkRecordField f) {
     builder.append(escaped(f.name));
     builder.append(" ");
     builder.append(getSqlType(f.type));
-    if (f.isOptional) {
+    if (f.defaultValue != null) {
+      builder.append(" DEFAULT ");
+      formatColumnValue(builder, f.type, f.defaultValue);
+    } else if (f.isOptional) {
       builder.append(" NULL");
     } else {
       builder.append(" NOT NULL");
     }
   }
 
+  static void formatColumnValue(StringBuilder builder, Schema.Type type, Object value) {
+    switch (type) {
```

Comment:
Good point. We don't support logical types elsewhere yet (in DDL `DbDialect.getSqlType()`, for insertion `PreparedStatementBinder.bindField`), so not worth starting here.

We should, though! Not sure if it's important for the initial release. I'll ticket it.